### PR TITLE
doc: updated SES import dialog description to 5.10c updates [DOC ONLY]

### PR DESCRIPTION
**Pull request by tejlmand**
The import dialog in SES has been updated to provide a list of projects
and boards to select for import.

This commit updates the getting started to reflect those changes.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_programming.rst [hunk 1]
```diff
@@ -34,11 +34,15 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
...
-   * **Build Directory** - the folder in which to run the build (automatically filled based on the board name, but you can specify a different directory)
-   * **Delete Existing CMakeCache.txt** - select this option to ensure that you are not building with an outdated build cache
+   * **Projects** - Select the sample that you want to work with
+   *  **Board Name** - Select the board that you want to work with
+   *  **Build Directory** - the folder in which to run the build (automatically filled based on the board name, but you can specify a different directory)
+   * **Clean Build Directory** - select this option to ensure that you are not building with an outdated build cache
+   * **Extended Settings** - Provides a field for specifying additional CMake arguments
+
+   .. note::
+	  To select a custom project, click click :guilabel:`...` next to projects list for opening the browse dialog and manual select a project.
```
Comment (bencefr) [inline doc/nrf/gs_programming.rst:44]: The double click is probably not intended. I also think that the second part of the sentence is not needed. 'To select a custom project, click :guilabel:`...` next to projects list.'

**Review by bencefr — COMMENTED** [review]
Is it ok not to describe anything about the 'nRFConnect SDK Release' and 'nRFConnect Toolchain Version'? Probably quite obvious, but they are there.

**Comment by tejlmand** [comment]
> Is it ok not to describe anything about the 'nRFConnect SDK Release' and 'nRFConnect Toolchain Version'?

Yeah, and I did consider to include those two fields.
But for a newcomer to NCS, that installs using getting started, then they should not need to care for those settings.

So I preferred to keep it simple, but can easily add those two fields.

@ru-fu thoughts ?

File: doc/nrf/gs_programming.rst (hunk 1 above)
Comment (tejlmand) [reply doc/nrf/gs_programming.rst]: Done.

**Review by ru-fu — CHANGES_REQUESTED** [review]
I'll add a commit with changes.

**Comment by ru-fu** [comment]
> > Is it ok not to describe anything about the 'nRFConnect SDK Release' and 'nRFConnect Toolchain Version'?
> 
> I did consider to include those two fields.
> But for a newcomer to NCS, that installs using getting started, then they should not need to care for those settings.
> 
> So I preferred to keep it simple, but can easily add those two fields.
> 
> @ru-fu thoughts ?

I'm very confused about those fields.
We support manual install as well, and afaik there is still no Toolchain manager for Linux, so not everybody will be using the Toolchain manager. What are you supposed to select in this case?

Also, what are the checkboxes underneath Projects and Board names? Are those filters?

**Comment by bencefr** [comment]
We intend to release the upcoming version of toolchain and toolchain manager for Linux as well.

**Review by bencefr — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Comment by tejlmand** [comment]
> I'm very confused about those fields.

Information given directly

**Comment by ru-fu** [comment]
> 
> 
> We intend to release the upcoming version of toolchain and toolchain manager for Linux as well.

Should this be the recommended installation for Linux in v1.4.0?
This isn't updated in the docs.
Oh, and I just see, it's not updated for macOS either ...

**Review by tejlmand — COMMENTED** [review]
Updated changes looks good.

**Comment by tejlmand** [comment]
> Docs are okay from my side now, but the commits need to be squashed before the PR can be merged.

Squashed.

**Pull request merged**